drivers/at86rf215/at86rf215_o-qpsk.c

```diff
@@ -222,18 +279,37 @@ static void _set_legacy(at86rf215_t *dev, bool high_rate)
                        | OQPSKC2_ENPROP_MASK);  /* enable RX of proprietary modes */
 }
 
+static inline void _set_ack_timeout_legacy(at86rf215_t *dev)
+{
+    dev->ack_timeout_usec = 54 * LEGACY_QPSK_SYMBOL_TIME_US;
+    DEBUG("[%s] ACK timeout: %"PRIu32" µs\n", "legacy O-QPSK", dev->ack_timeout_usec);
+}
+
 static void _set_ack_timeout(at86rf215_t *dev, uint8_t chips, uint8_t mode)
 {
-    dev->ack_timeout_usec = AT86RF215_ACK_PERIOD_IN_BITS * US_PER_SEC / _get_bitrate(chips, mode);
+    uint16_t symbols = _get_cca_duration_syms(chips)
+                     + _get_shr_duration_syms(chips)
+                     + 15   /* PHR duration */
+                     + _get_ack_psdu_duration_syms(chips, mode);
+
+    dev->ack_timeout_usec = _get_symbol_duration_us(chips) * symbols
+                          + 2 * AT86RF215_ATURNAROUND_TIME_SUN_US;
```

Comment:
why the 2?

Reply:
Must have been an error.

![Calculating PHY dependent MAC PIB values](https://user-images.githubusercontent.com/1301112/80007736-fcecf480-84c6-11ea-93ed-df28baee9e04.png)